### ISSUE TITLE
ttl.tensor.rsqrt operation breaks with low PCC on some ranges [Wormhole]

### ISSUE THREAD

**Issue by nemanjagrujic** [tenstorrent-metal/tt-metal#3523]
**ttl.tensor.rsqrt** operation breaks with  low PCC on some ranges.

**To Reproduce**
1. Checkout `main` branch
2. Run unit test `test_eltwise_rsqrt_in_depth.py` using the command:
`pytest tests/tt_eager/python_api_testing/non_working_unit_tests/grayskull/test_eltwise_rsqrt_in_depth.py`
[PS. Unit test is in grayskull subfolder since its the same issue on gray skull card]

**Expected behavior**
There are test cases presented in `test_eltwise_rsqrt_in_depth.py ` unit test which go through various ranges of input values and 126 of them  are expected to fail with low PCC error. 

Ranges swept:
* 1.4 - 1.5
* 1.5 - 1.6
... 
* 4.9 - 5.0
* 5.0 - 6.0
* 6.0 - 7.0
...
* 99.0 - 100.0

Test cases with most of those ranges fail with low PCC (<0.99)

**Comment by ruthreshx** [comment]
Hi @nemanjagrujic ,
I don't see the file name test_eltwise_rsqrt_in_depth.py from the path tests/tt_eager/python_api_testing/non_working_unit_tests/wormhole/test_eltwise_rsqrt_in_depth.py on the latest main.

Is the issue is still persistent?

**Comment by ruthreshx** [comment]
Hi @nemanjagrujic ,
I don't see the file name test_eltwise_rsqrt_in_depth.py from the path tests/tt_eager/python_api_testing/non_working_unit_tests/wormhole/test_eltwise_rsqrt_in_depth.py on the latest main.

Is the issue is still persistent?

**Comment by nemanjagrujic** [comment]
@ruthreshx Was deleted in the meantime (although it fails). Look in branch ngrujic/test_elt_rsqrt_in_depth. I recreated file from tag and checked (it still fails). Ill try to merge branch to main soon.

**Comment by nemanjagrujic** [comment]
@ruthreshx Unit test is now merged to main.

**Comment by ruthreshx** [comment]
Hi @nemanjagrujic ,
Rsqrt has been implemented using the Newton's method iteration. Hence the accuracy is too low was an expected one.
Between I could see the supported range is -10 to 10. I would suggest to fix **inc rate** as **0.5**. Generating the values between the low ranges will lies into low PCC.

```
detail::bind_unary_op_with_param(
            m_tensor, "rsqrt", &rsqrt,
            py::arg("fast_and_approx") = true,
            R"doc(Returns a new tensor with the reciprocal of the square-root of each of the elements of the input tensor ``{0}`` for the input range -10 to 10.)doc",
            R"doc("Indicate true for approx and fast mode; false for accurate and slow mode", "bool", "default of true")doc"
        );
```

**Comment by nemanjagrujic** [comment]
Tested, works now.
